openssl-dynamic/src/main/c/jnilib.c

```diff
@@ -243,8 +243,16 @@ static char* parsePackagePrefix(const char* libraryPathName, jint* status) {
         return NULL;
     }
 #ifdef _WIN32
+    char* tmpLibraryPathName = libraryPathName;
+
+    // replace \ with /
+    for(; *tmpLibraryPathName != '\0'; ++tmpLibraryPathName) {
```

Comment:
@Scottmitch I moved this here to ensure we always have `/` before call `netty_internal_tcnative_util_rstrchar `